h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -505,13 +509,11 @@ private int nextInstanceCounter(String algoName, NonBlockingHashMap<String, Inte
    * @param key (optional) model key
    * @param algo the Algo, e.g. {@link Algo#GBM}; used for validation, messages and for building the key if missing
    * @param parms the model builder params
-   * @param ignoreMaxModel (defaults to false) whether or not to skip the max_models validation
-   * @todo ignoreMaxModel param is a temp workaround to handle "exceptional" handling of SE,
-   *       it can be removed completely when we decide to consider SE as just another model
+   * @param ignoreLimits (defaults to false) whether or not to ignore the max_models/max_runtime constraints
```

Comment:
@seb-h2o What is the use of this new parameter -- in what case would it be useful to ignore the limits?

Reply:
This new param is only used for stacked ensemble building so it can be ensured the SE model is actually built.

Reply:
btw, not sure there's any good reason for this method to be public: if the user wants to retrain model previously trained by AutoML for example, I believe s·he can just use the model API.

Reply:
@seb-h2o Ok, let's make this private (if we can) so that people using Java won't try to use it.